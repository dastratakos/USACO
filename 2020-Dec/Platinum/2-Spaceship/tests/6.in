60 5 60
000110100111011111110011111110011011110101110111011011101111
110110111111000110101110010011011111011111111011111111110101
111111010110111100110110000111001101011011101110101001110110
000100010110100101110011111111111110011011010101001011011111
110001111011011011111000111011000111100111011101101110101010
110111101000001101111111001111111110111110011011100011100101
001111101110111011111110011110010011100101111000011011101111
110110101111011000111111010111111111000110000111111000011111
010101101001101101011011011111100111011111001110110011110101
110110011011111101000111011111110010110011110011011100010010
101110110010011001011111111011011011100110110100101110111011
001000111111100111101111001010000100110001011100111010011100
011111101101111011110101111011110101111011001110011010111111
101110111001011000011011111110011111010110011111111001101111
111110101111101111100110101111001111111110110101101111011000
101011111111111111111111110001111001010010111100111010111110
101001111001110101111111101101110000111011110000001101001001
100111101111100010111111111110001101010110000111011111011101
110111101011100001000111101111101011110111011001001111000010
011110010101011111101111111011111011100111011110011110110101
111011010111100111011111101011011101001001101010010101011011
011000111011111000101010110111111001111000001110111110101111
011001111101101111111011110111101111111011001111101111111111
011011101101101110111010111011111011101011100111101011110101
111110001011111111001011111011100010111000011010001011100000
100111111111110110110111001110110011000111100011101001110011
110111010111000101111101111100010011101101100111111000010110
111111000101111110011011110101011100011011101001100111111100
111110001101111011000111100011110111110011111111101100001011
111101110101111011111000100111010111111110111010111100111010
001100111010001100001010010001011011111111110111111101111100
111010000011110111011111011101000010101111101110011101011011
010101110010111111010011110011111000111111111001111111110101
001011111110011111100111001110001110011111011101011010010100
111011011101111101111101001111011011100110010100101110111111
000110111101111101111111001000111111011010111101011001011010
101110101110101111001001101110101111110011110111011111101001
001011010101100011011000000011111110101111111100111011111111
111111011001111111010101111011011110101001000110111111111110
101111010101011011111111100001111101111101101111111111001110
101111111100111111011111101000110001010110101001111100001101
111010010011101101011111111111011111111001110111001010111111
101111110111111111101111011101011001001111110111011011111000
111110111011010010001100101100111111110111111111110100111110
111111001011011111011011111111011111110011111110110111011001
101011111101100111100111111111110111101111111111001011001010
110001110111111011010101110111011011001101100110011001111011
110110101010001101001001101111101101101101011111101111111100
101111101001001111111111101111011001111111001001011100011111
011001011001110111000100100101111111001100110101011110111011
100111101100110111111111100111100101101110000111011000111111
101110101010000011011110101101110111110111010001010011011110
111111101100111011011011101011110110010011111011110110011110
011110111101111111011101100011110001111111110101011101100010
111001111111111000111110110111010111011100111101001101011010
110111101111111111101101100100110110011010010000010111101111
111111101101001001100001110011100111011111111111010111010011
100111111101110000001101110101011111111100101101111100100011
110001111100001011100001111011100110101100101001001111011101
111100111000101011100000001111101010011010110001101001110111
1 19 4 27
1 19 4 25
1 19 2 25
1 19 5 14
1 19 3 18
1 19 4 33
1 19 5 1
1 19 4 18
1 19 3 46
1 19 4 40
1 19 1 51
1 19 4 46
1 19 4 25
1 19 5 55
1 19 1 13
1 19 5 12
1 19 3 2
1 19 3 16
1 19 5 58
1 19 3 37
1 19 5 13
1 19 5 19
1 19 2 37
1 19 2 29
1 19 2 52
1 19 1 26
1 19 1 25
1 19 2 50
1 19 5 9
1 19 4 14
1 19 5 49
1 19 2 14
1 19 3 53
1 19 4 52
1 19 3 44
1 19 4 16
1 19 5 8
1 19 3 1
1 19 4 43
1 19 2 3
1 19 3 25
1 19 1 37
1 19 5 27
1 19 4 60
1 19 4 3
1 19 4 27
1 19 4 42
1 19 2 25
1 19 2 9
1 19 2 1
1 19 1 36
1 19 1 22
1 19 5 2
1 19 5 24
1 19 5 52
1 19 5 1
1 19 5 22
1 19 3 1
1 19 5 19
1 19 1 19
