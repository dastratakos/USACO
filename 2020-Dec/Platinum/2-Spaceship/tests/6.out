361570155
878706765
633634392
604767724
96497275
668578420
576283294
806208255
814916170
590594483
139294880
618924900
878706765
169919488
258696167
33404616
776625060
974525847
541176492
618366485
802094145
472547715
740115260
192146163
720061694
844420319
672811895
900970869
117323699
176809697
794803843
357134362
742963473
898298028
681463551
703272447
901785291
710120509
491610472
914953669
963614779
841777137
433296491
847751903
880628016
361570155
686708765
633634392
489497648
755843553
26465621
590817477
82589261
769883328
512444935
576283294
575893777
710120509
472547715
978484538
